commit 706b2be416246dd5923478d8a7ee015246f03e95
Author: Patrick Donnelly <pdonnell@ibm.com>
Date:   Thu Jan 30 17:05:25 2025 -0500

    mgr: use std namespace
    
    This C++ code was relying on the Client.h header to bring in these names from
    the std:: namespace. A subsequent commit plans to remove that header so add
    namespace qualifier now.
    
    Signed-off-by: Patrick Donnelly <pdonnell@ibm.com>

diff --git a/src/mgr/Mgr.cc b/src/mgr/Mgr.cc
index 26f93563517..9acd58248fb 100644
--- a/src/mgr/Mgr.cc
+++ b/src/mgr/Mgr.cc
@@ -115,7 +115,7 @@ void MetadataUpdate::finish(int r)
 
       if (daemon_state.exists(key)) {
         DaemonStatePtr state = daemon_state.get(key);
-	map<string,string> m;
+	std::map<string,string> m;
 	{
 	  std::lock_guard l(state->lock);
 	  state->hostname = daemon_meta.at("hostname").get_str();
@@ -143,7 +143,7 @@ void MetadataUpdate::finish(int r)
         }
         daemon_meta.erase("hostname");
 
-	map<string,string> m;
+	std::map<string,string> m;
         for (const auto &[key, val] : daemon_meta) {
           m.emplace(key, val.get_str());
         }
@@ -332,7 +332,7 @@ void Mgr::init()
        ++p) {
     string devid = p->first.substr(7);
     dout(10) << "  updating " << devid << dendl;
-    map<string,string> meta;
+    std::map<string,string> meta;
     ostringstream ss;
     int r = get_json_str_map(p->second, ss, &meta, false);
     if (r < 0) {
@@ -451,7 +451,7 @@ void Mgr::load_all_metadata()
     daemon_meta.erase("name");
     daemon_meta.erase("hostname");
 
-    map<string,string> m;
+    std::map<string,string> m;
     for (const auto &[key, val] : daemon_meta) {
       m.emplace(key, val.get_str());
     }
@@ -476,7 +476,7 @@ void Mgr::load_all_metadata()
     osd_metadata.erase("id");
     osd_metadata.erase("hostname");
 
-    map<string,string> m;
+    std::map<string,string> m;
     for (const auto &i : osd_metadata) {
       m[i.first] = i.second.get_str();
     }
@@ -742,7 +742,7 @@ bool Mgr::got_mgr_map(const MgrMap& m)
   std::lock_guard l(lock);
   dout(10) << m << dendl;
 
-  set<string> old_modules;
+  std::set<string> old_modules;
   cluster_state.with_mgrmap([&](const MgrMap& m) {
       old_modules = m.modules;
     });
